lib/addons/aws-for-fluent-bit/index.ts

```diff
@@ -0,0 +1,76 @@
+import { PolicyStatement } from '@aws-cdk/aws-iam';
+import { Construct } from "@aws-cdk/core";
+
+import { HelmAddOn, HelmAddOnUserProps } from "../helm-addon";
+import { ClusterInfo } from "../../spi/types";
+import { createNamespace } from "../../utils/namespace-utils";
+
+/**
+ * Configuration options for the FluentBit add-on.
+ */
+export interface AwsForFluentBitAddOnProps extends HelmAddOnUserProps {
+    /**
+     * Iam policies for the add-on.
+     */
+    iamPolicies?: PolicyStatement[]
```

Comment:
We need an issue to create a pattern for at least one case when this field is populated with policy per resource (e.g. access to CW or ES).